### DISCUSSION TITLE
tableWidget 有没有方法能查看每一行是不是隐藏状态？

### DISCUSSION THREAD

**Discussion by LiHongyuan1** [PyQt5/PyQt#161] [Q&A] [answered]
将 tableWidget 的数据做根据某些关键字进行删选。
把不包含关键字的 行 隐藏 self.tableWidget.setRowHidden(_, False)
把包含关键字的 行 显示 self.tableWidget.setRowHidden(_, True)
然后我在保存文件的时候想只保存显示的行的数据？ 但是似乎没有方法能够支持

**Comment by 892768447** [comment, marked as answer]
https://doc.qt.io/qt-5/qtableview.html#isRowHidden

self.tableWidget.isRowHidden(row) -> True/False

**Reply by LiHongyuan1** [reply]
thanks~
